xmtp_mls/src/groups/device_sync/consent_sync.rs

```diff
@@ -60,68 +50,58 @@ pub(crate) mod tests {
         amal_a.set_consent_states(&[consent_record]).await.unwrap();
 
         // Ensure that consent record now exists.
-        let syncable_consent_records = amal_a.syncable_consent_records(amal_a_conn).unwrap();
+        let syncable_consent_records = amal_a
+            .syncable_consent_records(amal_a.provider.conn_ref())
+            .unwrap();
         assert_eq!(syncable_consent_records.len(), 1);
 
         // Create a second installation for amal with sync.
-        let amal_b = ClientBuilder::new_test_client_with_history(&wallet, &history_sync_url).await;
+        let amal_b = amal_a.clone().await;
```

Comment:
is the clone an implicit new installation?

if i were writing a test with `Tester` I'm not sure I would have caught that, what if i'm sending amal_a to a separate thread and accidentally create another installation?

I feel like a function like `create_installation`/new_installation/init_installation or something along those lines would be more self-documenting

Reply:
Yeah, I agree - one of those things I would have eventually renamed. I'll think of a better name.

Reply:
Wait. Cloning a client creates a new installation?

Reply:
No, cloning a Tester creates a new installation. It's a helper struct for testing that wraps a client.